libnetwork/netavark/config_test.go

```diff
@@ -1353,6 +1365,7 @@ var _ = Describe("Config", func() {
 			Expect(logString).To(ContainSubstring("Network config name \\\"name_miss\\\" does not match file name \\\"name_missmatch.json\\\", skipping"))
 			Expect(logString).To(ContainSubstring("Network config \\\"%s/wrongID.json\\\" could not be parsed, skipping: invalid network ID \\\"someID\\\"", networkConfDir))
 			Expect(logString).To(ContainSubstring("Network config \\\"%s/invalid_gateway.json\\\" could not be parsed, skipping: gateway 10.89.100.1 not in subnet 10.89.9.0/24", networkConfDir))
+			Expect(logString).To(ContainSubstring("Network config \\\"%s/bad_metric.json\\\" could not be parsed, skipping: strconv.ParseUint: parsing \\\"-10004365\\\": invalid syntax"))
```

Comment:
```suggestion
			Expect(logString).To(ContainSubstring("Network config \\\"%s/bad_metric.json\\\" could not be parsed, skipping: strconv.ParseUint: parsing \\\"-10004365\\\": invalid syntax", networkConfDir ))
```
Also AFAIK we do not validate options on load from disk so this test will fail?, only network create makes the checks.

Reply:
Hm. Fair enough.

Reply:
I'll trim the invalid config test, I think you're right that it won't fail